scripts/prepare_sector_network.py

```diff
@@ -1175,7 +1175,7 @@ def add_industry(n, costs):
         n.loads.loc[nodes + co2_release, "p_set"].sum()
         * costs.at["oil", "CO2 intensity"]
         # - industrial_demand["process emission from feedstock"].sum()
-        / 8760
+        # / 8760

```

Comment:
Why is the 8760 division kicked out? To me it seems like `n.loads.loc[nodes + co2_release, "p_set"].sum()
        * costs.at["oil", "CO2 intensity"]` gives an annual value, hence it should be divided by 8760 if I am not wrong here. But probably the sum is over all nodes and not over the time dimension, hence kicking out the 8760 is a good bug catch